src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx

```diff
@@ -81,7 +83,7 @@ export const ContactInformation: React.FC<ContactInformationProps> = ({
           action: ActionType.consignmentSubmitted,
           submission_id: submission.id,
           user_id: me?.internalID,
-          user_email: me?.email ?? undefined,
+          user_email: isLoggedIn && me?.email ? me.email : submissionEmail,
```

Comment:
note: I changed this after abhiti's feedback: if a user is logged in we grab their email we have, and if they're not logged in then we grab the one they entered.